src/OrchardCore/OrchardCore.Application.Cms.Targets/OrchardCore.Application.Cms.Targets.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.ContentLocalization\OrchardCore.ContentLocalization.csproj" />
```

Comment:
None of these references should be in this group, move them with the other ones, ordered

Reply:
With privateassets = none?
And do I move also : <ProjectReference Include="..\..\OrchardCore\OrchardCore.Infrastructure\OrchardCore.Infrastructure.csproj" />

Reply:
Not Infrastructure, it's not a module, but it should probably already be referenced by something else and not necessary here

Reply:
> privateassets = none?

Yes for all modules and Infrastructure is certainly not needed.